containerimages/container-disk-images/customize-image.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -xe
+
+IMAGE_PATH=$1
+NEW_IMAGE=$2
+
+readonly CLOUD_CONFIG_PATH="user-cloud-config"
+readonly DOMAIN_NAME="provision-vm"
+readonly CLOUD_INIT_ISO="cloudinit.iso"
+
+# Create cloud-init user data ISO
+cloud-localds $CLOUD_INIT_ISO $CLOUD_CONFIG_PATH
+
+# Provision the VM by running could-init (ctrl+] to exit)
+virt-install \
```

Comment:
Would you need a virt-sysprep call afterwards?

Reply:
At first I used `virt-sysprep` but apparently it wont clean up machine-id as I would expect
I mean I created two VM's afterwards with the prepared image and both VMs had the same machine-id.

FYI I separated the process of customizing to different script in case we will come up with better method

Reply:
According to the man-page you have to specify `virt-sysprep --operations=machine-id`: http://libguestfs.org/virt-sysprep.1.html#machine-id

Reply:
btw, did you look at `virt-customize` as suggested by @EdDev ?

Reply:
Yes of course, I tried to use `virt-customize` on the image directly (no live VM) 
but there were things I couldn't do, for example:
- Install specific version of `kernel-modules` "dynamically" with `kernel-modules-$(uname -r)`.
- Creating secondary privileged used (that is not root) and set password.
- I also tried to run script with all necessary steps directly using `virt-customize --run prepareVM.sh`
but it didnt work, the script just wont run / stuck or doesn't do what it should (no idea why) :/

I will give `virt-customize` another chance maybe I missed something.

I have managed to do steps with `virt-sysprep`, I will update the cloud-config file in following commits.